.github/workflows/dispatch_release.yml

```diff
@@ -0,0 +1,17 @@
+name: Update homebrew
+
+on:
+  release:
```

Comment:
Can we check that we don't do this for releases marked as 'pre-release'?

Reply:
In theory that's what "released" is for: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#release